pkgs/development/tools/sikuli/default.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv, fetchurl, makeWrapper, utillinux, jre, jdk, opencv, tesseract, xdotool, wmctrl }:
+
+stdenv.mkDerivation rec {
+  name = "sikulix-${version}";
+  version = "1.1.0.20140219";
```

Comment:
This is from unversioned nightly release, that's why such strange version. Stable release is too old, i was unable to make it work.